### PR TITLE
Update style.css

### PR DESCRIPTION
**Pull request by CarlosZ96**
Minor CSS changes.

### CODE REVIEW

**Review by CarlosZ96 — COMMENTED** [review]
Hi @CarlosZ96.

There are a few issues you still need to work on to move on to the next project. But don't worry, you're just starting to get familiar with GitHub.


-**Your README file contains no information:** You will have to create a professional file that describes the contents of your project, you can use a template when creating your README file, it is very easy, you just have to section the template and replace with your information!


-**You are not using Linters**: 
In order for you to pass the code review, it is essential to use flashlights, if you are not familiar with this yet, just follow these small steps:

In the first commit of your feature branch create a .github/workflows folder and add a copy of [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) to that folder.
Remember to use the file linked above
Remember that .github folder starts with a dot.
Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.
If you think that change is necessary - open a [Pull Request in this repository](https://github.com/microverseinc/linters-config/blob/master/README.md#contributing) and let your code reviewer know about it.
When you open your first pull request, you should see the result of the GitHub actions checks.

Click on the Details link of each action to see the full output and the errors that need to be fixed.



-**You did not use correctly Git & GitHub Flow**: 
GitHub Flow is required for each of your projects, you can learn more about GitHub Flow at the following link: [https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/git_github_basics.md](url)

-**Update your stylesheet to resemble the requested result**:  You must correct the position of the different elements of the page, fonts, letter sizes, classes and ID required in the step.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @CarlosZ96 , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Acknowledged effort ✅   
- Gave Detailed comments ✅ 

## Required Changes ♻️

- [ ]  Great job on the review. You have left some great comments however there are a few things you might add to improve them. I would suggest you to use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments.
    - [ ] Also please make sure while you are using inline comments, you highlight the line or block of code that fits the context of the comment that you are making.
- [ ]  Please make sure to phrase the comments politely as suggestions instead of orders. For example:

* This is a polite suggestion: *"I noticed you have a missing README file: I would suggest you to create one using this templat.."*
* This reads as an order: *"Your README file contains no information: You will have to create a professional file..."*

- [ ] Please make sure to include an explanation, a "why", for each requested change. For example:

* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
* This request lacks a "why": *Why don't you use an article tag here instead of a div?*

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 1]
```diff
@@ -21,4 +21,5 @@
     align-items: center;
     color: black;
     font-family: 'Champions Light';
-  }
```
Comment (CarlosZ96) [inline css/style.css:24]: I see that already in line 24, after adding the font type, you finished your style sheet, I reviewed the final project and I think that you should add some things in your CSS file to be able to pass the challenge correctly, here are some suggestions that you can put to make your page look better and more similar to the final project:

-I think you can use the Background-color property to modify the background color of <main> tag and <figure> tag to make them more similar to the final project.

-You could use the padding:, margin: or text-align: properties to get them positioned as in the final requested project.

**Review by CarlosZ96 — COMMENTED** [review]
Hi @CarlosZ96.

Good job creating your first repository on GitHub!; however, there are a few issues you still need to work on to move on to the next project. But don't worry, you're just starting to get familiar with GitHub.


-I saw that you don't have a **README** file this file helps us to make our projects look more professional, so maybe you should create one, and you should always create it why in order to pass all the revisions of each project in Microverse, it should have a description of the project as a table of contents and then simply use a template, and you will only have to replace your data.

-I realized you don't have the files to use **Linters**: 
I think it's important that you create the files because in order for you to pass the code review, it is essential to use flashlights since it automatically reviews errors in the code and suggests how to fix it, if you are not familiar with this yet, just follow these small steps:

In the first commit of your feature branch, create a .GitHub/workflows folder, and add a copy of [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) to that folder.
Remember to use the file linked above
Remember that .github folder starts with a dot.
Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.
If you think that change is necessary - open a [Pull Request in this repository](https://github.com/microverseinc/linters-config/blob/master/README.md#contributing) and let your code reviewer know about it.
When you open your first pull request, you should see the result of the GitHub actions checks.

Click on the Details link of each action to see the full output and the errors that need to be fixed.


-**I saw that you did not use correctly Git & GitHub Flow**: 
GitHub Flow is required for each of your projects, you can learn more about GitHub Flow at the following link: [https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/git_github_basics.md](url)

-**Update your stylesheet to resemble the requested result**:  You must correct the position of the different elements of the page, fonts, letter sizes, classes and ID required in the step.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @CarlosZ96 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Very detailed comment instructions. ✔️ 
 - Good use of inline comments. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**